js/style.scss

```diff
@@ -1,143 +1,138 @@
 body {
   background-color: #eeeeee;
-  color: #777777;
+  font-family: sans-serif;
+}
 
-  .container > ul > li {
-    border: transparent 8px solid;
-    border-top: 0px;
-    border-bottom: 0px;
-  }
-  ul {
-    padding: 0;
-    .search {
-      background-color: white;
-      padding: 10px;
-      text-align: center;
-      input {
-        border-radius: 3px;
-        width: 70%;
-        height: 36px;
-        border: 1px solid #eeeeee;
-        &::placeholder {
-          padding: 10px;
-        }
-      }
+.container > ul > li {
+  border: transparent 8px solid;
+  border-top: 0px;
+  border-bottom: 0px;
+}
+ul {
+  padding: 0;
 
-      .fa-search {
-        background-color: #e67d22;
-        padding: 10px;
-        color: #fff;
-        font-size: 20px;
-      }
+  .search {
+    background-color: white;
+    padding: 10px;
+    text-align: center;
+
+    input {
+      border-radius: 3px;
+      width: 70%;
+      height: 36px;
+      border: 1px solid #eeeeee;
+      &::placeholder { padding: 10px; }
```

Comment:
Apply this rule from (https://codeguide.co/)
![image](https://user-images.githubusercontent.com/93534205/140617708-6899afab-756e-41c6-bdec-d0ef2157d815.png)

```suggestion
&::placeholder {
  padding: 10px;
}
```